nomt/src/page_cache.rs

```diff
@@ -146,11 +149,50 @@ struct CacheShard {
 }
 
 struct CacheShardLocked {
+    // storage for pages in the levels of the tree which we always cache.
+    fixed_level_cache: HashMap<PageId, CacheEntry, FxBuildHasher>,
```

Comment:
Nit: "Fixed" is ok, but I wonder if something like pinned would be slightly better?